graph paper.tex

```diff
@@ -1,8 +1,8 @@
 \documentclass{article}
 
-\usepackage[pattern=std]{gridpapers}
+\usepackage[pattern=hexup]{gridpapers}
 
-\begin{document}
+\begin{document}  
 \thispagestyle{empty}
-~
-\end{document}
+~ 
+\end{document} 
```

Comment:
Does this file need to change for this commit?

Reply:
No!